components/closed/as/Makefile

```diff
@@ -29,6 +29,7 @@ COMPONENT_ARCHIVE_HASH=	sha256:1758eb4721981d76c86dfe05b20d8dbccadc5938800554d2d
 
 include ../../../make-rules/prep.mk
 include ../../../make-rules/ips.mk
+include $(WS_MAKE_RULES)/forge.mk
```

Comment:
Changes in all these Makefiles will cause that all affected components will be rebuild and republished.  I suspect this is not a good idea because:

- it is likely that many such component builds will fail,
- there are missing `COMPONENT_REVISION` updates and so the built packages will not be consistent,
- there are also missing component metadata updates (`pkg5` files).

We should find other way to do this (prefereably by converting all components to use `common.mk`).

Reply:
I have yet to find a way to have them included in the metadata and me not needing to update the makefile. I could add forge.mk through shared-macros.mk but otherwise I cannot include forge.mk in all components that need metadata generation. For repology we need all component metadata relatively fast and not in a it will be eventually there.